docs/production-setup.md

```diff
@@ -271,7 +368,7 @@ Create a file called `docker-compose.yml` inside of the `zkevm-node` directory.
 > to the DB in the Explorer environment variable called DATABASE_URL
 > Remember to set the environment variable ETHEREUM_JSONRPC_HTTP_URL with the `zkEVM-Node` IP and PORT
 
-```docker-compose
+```yaml 
```

Comment:
The explorer expects the JSON RPC server to expose the debug endpoints and the previous configuration of the RPC is not exposing it.

Either we need to expose it or have a separate JSON RPC Server exclusive for the explorer that can expose the debug endpoints.

Since we suggest having a dedicated machine to it, maybe we should also have a different RPC server for this, as we do locally.

Reply:
>Either we need to expose it or have a separate JSON RPC Server exclusive for the explorer that can expose the debug endpoints.

@arnaubennassar should we do this?

Reply:
I've gone ahead and added a separate zk node running the RPC component with the debug stuff in it. Only for explorer use. @tclemos